src/sickle.ts

```diff
@@ -187,12 +211,15 @@ class Annotator {
     this.emit(';\n');
   }
 
-  private writeNode(node: ts.Node) {
+  private writeNode(node: ts.Node, skipComments?: boolean) {
```

Comment:
how about `skipComments: boolean = false` instead, so that we keep `undefined` out of the runtime.